fjord/base/models.py

```diff
@@ -16,6 +18,47 @@ class Profile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
 
 
+class ListField(models.TextField):
+    description = _lazy('List of values')
+
+    __metaclass__ = models.SubfieldBase
+
+    def __init__(self, *args, **kwargs):
+        super(ListField, self).__init__(*args, **kwargs)
+
+    def to_python(self, value):
+        if not value:
+            value = []
+
+        if isinstance(value, list):
+            return value
+
+        try:
+            # This takes a unicode and reconsistutes it into Python base
+            # types.
+            return ast.literal_eval(value)
+        except Exception as exc:
```

Comment:
I don't know if SyntaxError is the only possible exception that can be thrown here. Regardless, I think I want to treat all exceptions as ValidationErrors, so seems better to keep it like this.